package.json

```diff
@@ -9,6 +9,7 @@
   "bugs": {
     "url": "https://github.com/ProjectOpenSea/stream-js/issues"
   },
+  "type": "module",
```

Comment:
Is this something that's needed for a package like an SDK?

Reply:
This might break usage in commonjs code (nodejs). I would not add right now, unless this is properly tested (by publishing a rc and trying to consume it in a node and browser environment)